### DISCUSSION TITLE
Active cell - Start editing with symbol key

### DISCUSSION THREAD

**Discussion by abennett418** [nick-keller/react-datasheet-grid#138] [Feature requests]
Hi,

Currently, if you have a cell active but you are not yet editing, you can start typing and it automatically starts editing. This however is limited and does not allow most symbols. Is there is a reason for only allowing a limited set?

I require the ability to auto start editing if the user inputs `=`, `<` or `>`. However, I do not see any reason why the rest of the symbols should not also be supported.

The [current regex](https://github.com/Equify/react-datasheet-grid/blob/master/src/components/DataSheetGrid.tsx#L1412) is `/^[a-zA-Z0-9 ,.+-]$/`

Some possible solutions:

- Update expression to be more broad: `/^[ -~]$/`
- Update expression to explicity include all symbols: ``/^[a-zA-Z0-9 !@#$%^&*()_+-=\[\]{}:;'",./<>?\\|~`]$/``
- Allow user to pass through desired express as prop

**Comment by nick-keller** [comment]
Hey @abennettHS ,
Available in 4.1.1

**Reply by abennett418** [reply]
Awesome, thanks Nick!
